config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-k8s-triage-robot.yaml

```diff
@@ -290,19 +290,9 @@ periodics:
       - --token=/etc/github-token/token
       - --endpoint=http://ghproxy.test-pods.svc.cluster.local
       - |-
-        --comment=The Kubernetes project has merge-blocking tests that are currently too flaky to consistently pass.
+        --comment=Retesting failed PR that otherwise appears ready for merge.
 
-        This bot retests PRs for certain kubernetes repos according to the following rules:
-        - The PR does have any `do-not-merge/*` labels
-        - The PR does not have the `needs-ok-to-test` label
-        - The PR is mergeable (does not have a `needs-rebase` label)
-        - The PR is approved (has `cncf-cla: yes`, `lgtm`, `approved` labels)
-        - The PR is failing tests required for merge
-
-        You can:
-        - Review the [full test history](https://prow.k8s.io/pr-history/?org={{.Org}}&repo={{.Repo}}&pr={{.Number}}) for this PR
-        - Prevent this bot from retesting with `/lgtm cancel` or `/hold`
-        - Help make our tests less flaky by following our [Flaky Tests Guide][1]
```

Comment:
(commenting with the only context from original PR adding this text - https://github.com/kubernetes/test-infra/pull/23094)

I agree with bringing the "call to action to fix flaky test" right up there, but rest text still feels important to keep IMO.

@cblecker @MadhavJivrajani - what do you think?

Reply:
I've had folks ask why the retester isn't doing it's job before, and it's typically because "oh it isn't lgtm'd yet" or "it has do-not-merge/work-in-progress". I'm fine shortening this down, but can we leave a breadcrumb/link?

I do feel having the "prevent this bot from retesting.." bit is important enough to keep here too, IMO

Reply:
This snippet isn't exactly accurate... it isn't in sync with the config, and if people ask, can't we just link them to some docs or the configuration? For every one occasional user that does ask, we post a massive wall of text that is just ignored by everyone else making all of the comments ~useless.

Reply:
I also don't think "document in detail via inline comment" is a reasonable approach.

Reply:
eesh https://github.com/kubernetes/test-infra/issues/35859